### ISSUE TITLE
Allow `XGBattle_SP` to return the number of units in `eTeam_One` and `eTeam_Two`

### ISSUE THREAD

**Issue by E3245** [X2CommunityCore/X2WOTCCommunityHighlander#616]
It would be useful to check the number of units in `eTeam_One` and `eTeam_Two` in Kismet, however, there's no way to do this since `XGBattle_SP` does not have a function to return the number of `XGAIPlayer_TeamOneDefault` and `XGAIPlayer_TeamTwoDefault` respectively.

**Comment by E3245** [comment]
In addition to this change, `SeqAct_GetAllUnitsOnTeam` and `SeqAct_GetUnitCounts` does not have support for `eTeam_One` and `eTeam_Two` because of this issue.

**Comment by Xymanek** [comment]
I don't think we can edit class layout of `SeqAct`s as that will break cooked maps

**Comment by E3245** [comment]
> I don't think we can edit class layout of `SeqAct`s as that will break cooked maps

Can we at least do the first part then? I could always just write my own SeqAct.
